Jenkinsfile

```diff
@@ -7,20 +7,41 @@ pipeline {
         stage('Checkout'){
             steps {
                 sh """
-                    git clone https://github.com/couchbaselabs/${env.PRODUCT}.git
-                    pushd ${env.PRODUCT}
+                    #!/bin/bash
+                    set -e
+                    shopt -s extglob dotglob
+
+		    # move PR related repo to tmp folder
+		    mkdir tmp
+                    mv !(tmp) tmp
+                    git clone https://github.com/couchbaselabs/${env.PRODUCT}.git --branch $CHANGE_TARGET
+
+		    # restructure folders
+		    mv couchbase-lite-ios-ee/* .
+		    mv tmp/* couchbase-lite-ios
+
+		    # update the lite-core-EE
+		    pushd couchbase-lite-core-EE
+		    git submodule update --init ./
```

Comment:
couchbase-lite-core-EE doesn't have any submodules so we can remove this section entirely. If couchbase-lite-core-EE starts to have submodule, we can add this.

Reply:
if not updated the submodule, then we will get this error
http://mobile.jenkins.couchbase.com/blue/organizations/jenkins/_cbl_github_pipeline%2Fcouchbase-lite-ios/detail/PR-2561/8/pipeline

Reply:
It really doesn't have submodules so `git submodule update --init ./` will do nothing. Please double check.